### PR TITLE
IDVA5-658-error-messages

### PR DESCRIPTION
**Pull request by abhiahirv1**
Setting up Error Messages on What is your Role? screen when no option is selected

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: src/test/src/controllers/limitedWhatIsYourRoleController.test.ts
Comment (AmritaSasikumar-CH) [inline src/test/src/controllers/limitedWhatIsYourRoleController.test.ts:1]: Please can you add a check for error message returned and also add test for sole trader role

**Review by sardhani-ch — APPROVED** [review]
please address Amrita comments. LGTM otherwise

File: src/test/src/controllers/limitedWhatIsYourRoleController.test.ts
Comment (abhiahirv1) [reply src/test/src/controllers/limitedWhatIsYourRoleController.test.ts:1]: Done

**Review by AmritaSasikumar-CH — APPROVED** [review]
Approved

**Review by sardhani-ch — APPROVED** [review]

**Pull request merged**